datalab/bigquery/_dialect.py

```diff
@@ -36,6 +36,12 @@ def set_bq_dialect(self, bq_dialect):
     """ Set the default BigQuery SQL dialect"""
     if bq_dialect in ['legacy', 'standard']:
       self._global_dialect = bq_dialect
+      try:
+        from google.datalab import Context as new_context
+        new_context.default().config['bigquery_dialect'] = bq_dialect
```

Comment:
we should remove `bigquery_dialect` everywhere, we don't support legacy SQL anymore in the new namespace.